etc/skydive.yml.default

```diff
@@ -163,6 +163,14 @@ analyzer:
   replication:
     # debug: false
 
+  table:
```

Comment:
I would prefer to use the flow section that is already existing:
https://github.com/skydive-project/skydive/blob/97921a7961f0bcead5d6fded82ce97cb501c7d2e/etc/skydive.yml.default#L445

the sub section could be `application_timeout` as we already have `application_ports`.